settings/global.mdx

```diff
@@ -62,6 +62,10 @@ settings. Learn more about the [properties](#properties) or from an
   </Expandable>
 </ResponseField>
 
+<ResponseField name="theme" type={'"venus" | "quill" | "prism"'}>
```

Comment:
I wonder if it would be useful to provide context (inline or otherwise) on when someone might want to use one of the three themes? maybe a condensed version of the descriptions we had in the blog post?